### PR TITLE
Defines a new PolicyOptions class and corresponding proto.

### PR DESCRIPTION
**Pull request by csilvestrini**
This is used to customise the policy enforcement algorithm. Customers will define their own config textproto file and use that when invoking the policy algorithm.

Currently this just has the storeMap field, but we can add things in future (e.g. whitelist of particle specs).

### CODE REVIEW

File: java/arcs/core/policy/PolicyOptions.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+/** Config options for policy enforcement. */
+data class PolicyOptions(
+    /**
+     * Maps from store ID to the schema name of the type it stores. Indicates all of the stores
+     * protected by policy
+     *
+     * Temporary measure until we have a proper way to designate these stores.
+     */
+    val storeMap: Map<StoreId, String>
```
Comment (bgogul) [inline java/arcs/core/policy/PolicyOptions.kt:13]: We will also need a map from `EgressType` to particle class? Is the plan to do it in a separate PR?

Comment (csilvestrini) [reply java/arcs/core/policy/PolicyOptions.kt:13]: I wasn't sure exactly what the mapping there was (1:1? 1:N?) so I thought I would leave it until we needed it and knew exactly what it was.
